src/apps/intel/intel_app.lua

```diff
@@ -116,6 +148,11 @@ function Intel82599:push ()
       end
    end
    self.dev:sync_transmit()
+   if self.dev.txstats then
```

Comment:
Makes sender, honestly I had/have trouble getting a good understanding of Intel10G stat registers code. Could definitely use a thorough review with SHM stats in mind (e.g. get rid of `Intel10G.dev.get_rx/tx_stats()` altogether). In this PR I attempted to be make mostly non-disruptive (light bolt-on) changes.